.github/workflows/build-test-rust.yml

```diff
@@ -103,12 +109,19 @@ jobs:
         with:
           workspaces: polars
 
+      - name: Use sccache
+        uses: mozilla-actions/sccache-action@v0.0.3
+
       - name: Run tests
         env:
           RUSTFLAGS: -C debuginfo=0  # Do not produce debug symbols to keep memory usage down
+          SCCACHE_GHA_ENABLED: "true"
+          RUSTC_WRAPPER: "sccache"
```

Comment:
We can probably define this at the top level, rather than for each step, right? Or is it important that these are NOT set for some other steps?

Reply:
I think this should be fine, will do